init.js

```diff
@@ -1,5 +1,137 @@
 module.exports = {
+  //TASK 1
   sum(a, b) {
-    
+    if (typeof a === 'string' || typeof b === 'string') {
```

Comment:
What about other data types?

Reply:
@nadeemcite  the variable is checked only for string data types